content/doc/book/installing/_jenkins-command-parameters.adoc

```diff
@@ -109,23 +106,22 @@ to use an existing certificate for HTTPS:
 
 === Using HTTP/2
 
-The link:https://tools.ietf.org/html/rfc7540[HTTP/2 protocol] allows web servers to reduce latency over encrypted connections by pipelining requests, multiplexing requests, and allowing servers to push in some cases before receiving a client request for the data.
+The link:https://tools.ietf.org/html/rfc7540[HTTP/2 protocol] allows web servers to reduce latency over encrypted connections by pipelining requests, multiplexing requests, and allowing servers to push, in some cases, before receiving a client request for the data.
 The Jetty server used by Jenkins supports HTTP/2 with the addition of the Application-Layer Protocol Negotiation (ALPN) TLS extension.
 The ALPN TLS extension is connected to the specific Jetty version and has specific requirements depending on the Java version.
 
-Note that enabling HTTP/2 implicitly enables TLS even if no HTTPS port is set, and as of Jenkins 2.339 (which uses Winstone 5.23) you have to also specify an HTTPS key store file.
+Note that enabling HTTP/2 implicitly enables TLS even if no HTTPS port is set, and as of Jenkins 2.339, which uses Winstone 5.23, you have to also specify an HTTPS key store file.
 
-==== Java 11, Java 8u252, and later
+==== Java 11
 
-IMPORTANT: Starting with weekly release line *2.357*, Java 11 will be required.
-The Long Term Support release line will not require Java 11 until September, at which point Java 8 support will be deprecated.
+IMPORTANT: Starting with weekly release line *2.357* and Long Term Support release line *2.361.1*, Java 11 is required.
```

Comment:
Similar to my previous comment, perhaps we should add "or 17" to verbiage like this for consistency. Feel free to ignore if you disagree.

Reply:
Definitely going to add it there as well, and make sure there is consistency throughout.